portal/server/nunjucks-configuration/index.js

```diff
@@ -17,6 +18,11 @@ const configureNunjucks = (opts) => {
 
   const nunjucksEnvironment = nunjucks.configure(appViews, opts);
 
+  nunjucksEnvironment.addGlobal(
+    'CONTACT_US_EMAIL_ADDRESS',
+    process.env.CONTACT_US_EMAIL_ADDRESS || 'DigitalService.TradeFinance@ukexportfinance.gov.uk',
```

Comment:
Object deconstruct `process.env` please.

Reply:
I think it is safe to remove `'DigitalService.TradeFinance@ukexportfinance.gov.uk'` as this will always be present.